### DISCUSSION TITLE
B.3.5. EKG/Platform Team - EKG/Maturity

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#82] [Page Comments]
# B.3.5. EKG/Platform Team - EKG/Maturity

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity.ekgf.org/pillar/organization/delivery-management/ekg-platform-team/

**Comment by jgeluk** [comment]
Let's call this "Center of Excellence" (for the EKG), any other suggestions?
